docs/_bugs/_resolved/BUG-001-float-equality-dominant-points.md

```
@@ -0,0 +1,44 @@
+**Status**: Resolved (Already Fixed)
+**Resolved**: 2026-01-03
+**Resolution**: Code already contains fix at `selective_prediction.py:388-397` with `epsilon = 1e-10`
+**Created**: 2026-01-03
+**File**: `src/ai_psychiatrist/metrics/selective_prediction.py`
+
+## Description
+
+The `_compute_dominant_points` function implements a Monotone Chain algorithm to find the lower convex hull of the risk-coverage curve. It uses a cross-product check to determine orientation:
+
+```python
+def _cross_product(o, a, b):
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)